compiler/rustc_target/src/spec/apple_base.rs

```diff
@@ -284,6 +284,22 @@ fn ios_lld_platform_version() -> String {
     format!("{major}.{minor}")
 }
 
+pub fn mac_catalyst_deployment_target() -> &'static str {
+    // Consider making this user-changeable in the future.
+    "14.0"
```

Comment:
QQ: Could we re-use the value from environment variable `IPHONEOS_DEPLOYMENT_TARGET`?

I'm not sure about this. Found `IPHONEOS_DEPLOYMENT_TARGET` in the same file (apple_base.rs).